.travis.yml

```diff
@@ -34,12 +34,24 @@ jobs:
       env:
```

Comment:
Can't comment on a line that's not changed, but line 29 above seems wrong too: `name: "Compile all code with 2.13"`

Reply:
Otherwise, LGTM